codenames-domain/src/main/java/com/codenames/services/GameService.java

```diff
@@ -0,0 +1,101 @@
+package com.codenames.services;
+
+import com.codenames.dto.RoomDto;
+import com.codenames.exceptions.RoomLimitIsOver;
+import com.codenames.models.game.CodeNamesGame;
+import com.codenames.models.game.User;
+import com.codenames.models.room.Room;
+import com.codenames.models.game.Player;
+import com.codenames.properties.CodeNamesGameProperties;
+import com.codenames.properties.DefaultMessagePathProperties;
+import com.jjerome.dto.Request;
+import com.jjerome.models.MessageSender;
+import jakarta.annotation.PostConstruct;
+import lombok.RequiredArgsConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.util.Random;
+
+@Service
+@RequiredArgsConstructor
+public class GameService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(GameService.class);
+
+    private final Random random = new Random();
+
+    private final CodeNamesGameProperties gameProperties;
+
+    private final MessageSender messageSender;
+
+    private final PlayerService playerService;
+
+    private final RoomService roomService;
+
+    private final DefaultMessagePathProperties defaultMessagePathProperties;
+
+    private int minRoomID;
+
+    private int maxRoomID;
+
+    private int roomLimit;
+
+    @PostConstruct
```

Comment:
Put this in a constructor instead (https://levelup.gitconnected.com/stop-using-postconstruct-in-your-java-applications-2a66fb202cb8)